styleguide/_themes/derek/scss/components/_rug.scss

```diff
@@ -0,0 +1,12 @@
+.rug{
+  .rugSubhead{
+    font-size: 37px;
+    line-height: 24px;
+  }
+  &, *{
+    @extend .white;
+  }
+  .contactLink{
```

Comment:
Hey @amberRrucker thanks for reading/implementing the MaintainableCSS style here. We are adding `sass-lint` in PR #105 and default configs force lowercase-hyphenated class naming form. We just talked about it and a group and are in agreement that we prefer lowercase-hyphenated to be more `sass-lint` compliant, and that is the only tweak we see necessary to make to MaintainableCSS for our needs.

Cool?